integration_test/multi_tentant_test/multi_tenant_test.go

```diff
@@ -233,7 +233,7 @@ func testMultiTenantByAppType(t *testing.T, appType string) {
 	)
 
 	select {
-	case ack := <-etcdContainer.Client.Watch(ctx, "test-ack/1"):
+	case ack := <-etcdContainer.Client.Watch(ctx, "test-ack-1/1", clientv3.WithRev(1)):
```

Comment:
@fracasula is you start watching from etcd without specifying a revision, any previously set keys (before you started watching) are not returned, unless you specify the minimum revision you want to receive

Reply:
Oh yes, this makes a lot of sense, thanks! :+1: